apps/api/resources/api/config.ttl

```diff
@@ -30,6 +30,14 @@ PREFIX knossos: <https://hypermedia.app/knossos#>
           code:link <file:apps/api/lib/auth.js#basic> ;
         ] ;
     ],
+    [
+      schema:name "before" ;
```

Comment:
I'm curious how is the order of the middleware defined?

Reply:
Currently there is none